File: 1657-determine-if-two-strings-are-close/1657-determine-if-two-strings-are-close.py
class Solution:
    def closeStrings(self, word1: str, word2: str) -> bool:
        w1 = Counter(word1)
        w2 = Counter(word2)

        if  w1.keys() != w2.keys():
            return False
            
        if sorted(w1.values()) != sorted(w2.values()):
            return False

        return True

File: 1657-determine-if-two-strings-are-close/1657-determine-if-two-strings-are-close.rb
# @param {String} word1
# @param {String} word2
# @return {Boolean}
def close_strings(word1, word2)
    w1 = word1.chars.tally
    w2 = word2.chars.tally

     w1.keys.sort == w2.keys.sort && w1.values.sort == w2.values.sort
end